### ISSUE TITLE
The license of the VisualScriptEngineWxWidgets issue

### ISSUE THREAD

**Issue by asmwarrior** [kovacsv/VisualScriptEngineWxWidgets#2]
Hi, the core engine(VisualScriptEngine) is in MIT license. While this project is in GPL license.

Do you mean that for developers who using wxWidgets, they have to open their project's source code if they use VisualScriptEngineWxWidgets?

I mean could this project have some permissive license, such as LGPL or MIT?

Thanks.

**Comment by kovacsv** [comment]
VisualScriptEngineWxWidgets is GPL, because it depends on wxWidgets that comes with a GPL license. As far as I know you can't redistribute something that uses wxWidgets under MIT. Of course it's possible that I'm wrong, but according to my knowledge it is not possible.

**Comment by asmwarrior** [comment]
Hi, wxWidgets library is **NOT** licensed under GPL.

See: [Licence - wxWidgets](https://www.wxwidgets.org/about/licence/)

> The wxWindows Library Licence is essentially the L-GPL (Library General Public Licence), with an exception stating that derived works in binary form may be distributed on the user’s own terms. This is a solution that satisfies those who wish to produce GPL’ed software using wxWidgets, and also those producing proprietary software.

**Comment by kovacsv** [comment]
Yeah, probably you are right. I'm not really good with licenses :) Thanks for reporting this, I'll check the possibilities.

**Comment by edgarbarney** [comment]
Still waiting for the updates.
